x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/AttributeSet.java

```diff
@@ -185,4 +183,4 @@ public int hashCode() {
     public String toString() {
         return delegate.keySet().toString();
     }
-}
\ No newline at end of file
+}
```

Comment:
Aaah. It seems that IDEA rereads the settings from `.editorconfig` all the time.